java/src/main/java/com/cybersource/ws/client/Client.java

```diff
@@ -34,6 +34,7 @@
 import java.util.HashMap;
 import java.util.Map;
 import java.util.Properties;
+import java.util.concurrent.ConcurrentHashMap;
 
 /**
```

Comment:
add all new properties in readme(there are 2 readme files, do in both).
add those properties with default values in cysb or test_cybs.properties file